doc/iterations/2025.1/add_support_of_non_synchronized_tables.adoc

```diff
@@ -0,0 +1,40 @@
+= Add support of non synchronized tables
+
+== Problem
+
+Currently, tables are synchronized regarding the semantic elements found in the model.
+The table defines, inside its row description, what kind of elements the table handles and how they are retrieved.
```

Comment:
Rows and columns should share the same capabilities. This shape should not be specific to rows.